app/scripts/controllers/eventRegistrations.js

```diff
@@ -93,6 +93,32 @@ angular.module('confRegistrationWebApp')
       $scope.queryParameters.block = visibleBlocks;
     };
 
+    // Set initial built in Display Column values
+    var builtInColumnsVisible = {
+      Email: false,
+      Started: false,
+      Completed: false
+    };
+
+    // Used on Display Columns menu click to hide built in columns
+    $scope.hideColumn = function(columnName) {
+      for (var key in builtInColumnsVisible) {
+        if (columnName == key) {
+          if (builtInColumnsVisible.hasOwnProperty(key)) {
+            //alert(key + " -> " + builtInColumnsVisible[key]);
+            if ($scope['checked' + columnName]) {
+              $scope['checked' + columnName] = false;
+              $scope['show' + columnName] = false;
+            }
+            else {
+              $scope['checked' + columnName] = true;
+              $scope['show' + columnName] = true;
+            }
+          }
+        }
+      }
+    }
```

Comment:
jshint (JS code style checker) wants a semicolon here and is causing the [travis build](https://travis-ci.org/CruGlobal/conf-registration-web/builds/144045874#L803-L804) to fail